.gitignore

```diff
@@ -0,0 +1 @@
+hammerspoon/hyper-apps.lua
```

Comment:
@bkeepers: Thanks for the review. :bow:

3db82009ea151884cdd94832206c2d801932c98b updates the logic to first look for your custom Hyper Mode mappings in `hammerspoon/hyper-apps.lua`. That file is ignored by Git, and you should be able to modify it without Git showing that it's been modified. If `hammerspoon/hyper-apps.lua` doesn't exist, we fall back to the default mappings in `hammerspoon/hyper-apps-defaults.lua`.

Wanna try this out and see if works for you?